### DISCUSSION TITLE
请问wifi6设备连接后上不了网的问题解决了吗？

### DISCUSSION THREAD

**Discussion by smith97318** [Boos4721/openwrt#74] [Q&A/答疑解惑] [answered]
之前发现用wifi6协议连接的设备，能连上wifi，但没有网络，wifi5的设备正常使用。
请问一下，现在这个问题解决了吗？如果解决了，解决这个问题的commit是哪个？

**Comment by betaxab** [comment]
试过了昨天的版本，还没有修复。我的 iPhone 连上了没有网，Android WiFi6 连都连不上。

**Reply by Boos4721** [reply]
Check yourself.

**Comment by zjwxh** [comment]
我也是同样的问题，红米A6编译出来，连接到5G之后 小米10手机不能上网，但是笔记本又可以，实在莫名其妙。用大佬这个源码编译非常不错，就可惜5G这上网问题搞不定

**Comment by Boos4721** [comment, marked as answer]
![iShot2022-02-17 13 15 59](https://user-images.githubusercontent.com/43845654/154410280-5159e53b-c472-42e7-97bc-46d4414af9a3.jpg)

**Reply by Boos4721** [reply]
MacBookPro 2019 and iPhone 11

**Comment by zjwxh** [comment]
今天编译的AX6固件5G网络连接之后可以上网了
